backend/test/services/schema.spec.ts

```diff
@@ -1,20 +1,21 @@
 import { describe, expect, test, vi } from 'vitest'
 
-import { createSchema, findSchemasByKind } from '../../src/services/v2/schema.js'
+import { createSchema, getSchemaById, getSchemasByKind } from '../../src/services/v2/schema.js'
 import { testModelSchema } from '../testUtils/testModels.js'
 
 const mockSchema = vi.hoisted(() => {
   const mockedMethods = {
     save: vi.fn(),
     deleteOne: vi.fn(),
     find: vi.fn(() => ({ sort: vi.fn(() => ['schema-1', 'schema-2']) })),
+    findOne: vi.fn(),
   }
 
   const Schema: any = vi.fn(() => ({
     save: mockedMethods.save,
   }))
   Schema.find = mockedMethods.find
-  Schema.deleteOne = mockedMethods.deleteOne
+  ;(Schema.findOne = mockedMethods.findOne), (Schema.deleteOne = mockedMethods.deleteOne)
```

Comment:
What is going on here?!  Put it on multiple lines!

Reply:
Caused by running npm run style, not sure why